lib/themes/dosomething/paraneue_dosomething/templates/reportback/reportback-permalink.tpl.php

```diff
@@ -18,7 +18,7 @@
 <?php if ($is_current): ?>
   <?php echo $copy_vars['owners_rb_subtitle'] ?>
   <?php echo $copy_vars['owners_rb_scholarship'] ?>
-  <?php echo $reportback->caption ?>
+  <?php echo check_plain($reportback->caption) ?>
```

Comment:
I believe this is the page @angaither is working on, which is hidden behind a flag on the DS Reportback config page.